shared/js/background/trackers.es6.js

```diff
@@ -98,6 +99,15 @@ function isTracker (urlToCheck, thisTab, request) {
     return false
 }
 
+// Check for reason to block the tracker
+// And add it
+// For now it just adds "beacon" if that's the tracker type
+function checkReason (trackerObj) {
```

Comment:
Since other nearby functions may also assign a reason may I suggest that this name is too generic. In this case I would probably just write this short bit of code inline above on line 91. You've already factored out `getBeaconName()` so it's pretty short. I think it is not likely to expand to cover more cases here since the other functions pretty clearly need to be refactored.

Reply:
I mostly created a function so it'd be easier to unit test - which is also why it needs to return something. I can change this though.

Reply:
Ok, then I would just make the name more specific. But this is so small a function, and the reason property is set in multiple places, maybe unit testing `getBeaconName()` is sufficient?

Reply:
Yeah, that's what I ended up doing. Thanks!